packages/mobile/src/components/social-button/SocialButton.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useMemo } from 'react'
+
+import type { ButtonProps } from 'app/components/core'
+import { Button } from 'app/components/core'
+import { makeStyles } from 'app/styles'
+
+export type SocialButtonProps = ButtonProps
+
+const useStyles = makeStyles(({ spacing }) => ({
+  button: {
+    padding: spacing(3),
+    height: 64
+  },
+  text: {
+    fontSize: 20
+  },
+  icon: {
+    height: 20,
```

Comment:
I know annoying, but we can use spacing

Reply:
How do we feel about using spacing for height/width dimension, like in this icon case? Vs just using for spacing/layout stuff